app/views/spree/components/layout/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="header" role="banner">
+  <div class="header_left">
```

Comment:
as we said IRL, let's try to use a more semantic class name here. left/center/right are information about the style of the elements.